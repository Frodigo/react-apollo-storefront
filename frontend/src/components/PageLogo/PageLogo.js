import React from 'react';


const PageLogo = () => {
    return <a href="/">React Apollo</a>
};


export default PageLogo;
