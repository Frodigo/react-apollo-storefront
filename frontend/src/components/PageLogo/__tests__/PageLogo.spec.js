import React from 'react';
import { render } from '@testing-library/react';

import PageLogo from '../PageLogo';

test('PageLogo renders correctly', () => {

    const { container } = render(<PageLogo />);

    expect(container).toMatchSnapshot();
});

