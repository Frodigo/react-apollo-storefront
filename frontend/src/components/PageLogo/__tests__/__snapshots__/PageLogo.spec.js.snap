// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLogo renders correctly 1`] = `
<div>
  <a
    href="/"
  >
    React Apollo
  </a>
</div>
`;
